nexus-repository-helm/src/main/java/org/sonatype/repository/helm/internal/hosted/HelmHostedRecipe.groovy

```diff
@@ -163,6 +170,20 @@ class HelmHostedRecipe
     )
   }
 
+  static Matcher provenanceMethodMatcher(final String... httpMethods) {
+    LogicMatchers.and(
+        new ActionMatcher(httpMethods),
+        tokenMatcherForExtensionAndName('tgz.prov'),
```

Comment:
tgz.prov should be stored as constant ion HelmPathUtil

Reply:
It is a good idea, but this constant will be used only once. By the way, Other formats have the same implementation.